[2024-10-02T18:59:16.867+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T18:59:16.896+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-15T00:00:00+00:00 [queued]>
[2024-10-02T18:59:17.104+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-15T00:00:00+00:00 [queued]>
[2024-10-02T18:59:17.104+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T18:59:17.122+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): carregar> on 2024-01-15 00:00:00+00:00
[2024-10-02T18:59:17.130+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=6239) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T18:59:17.132+0000] {standard_task_runner.py:63} INFO - Started process 6249 to run task
[2024-10-02T18:59:17.131+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'carregar', 'scheduled__2024-01-15T00:00:00+00:00', '--job-id', '1291', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp6t87ehel']
[2024-10-02T18:59:17.133+0000] {standard_task_runner.py:91} INFO - Job 1291: Subtask carregar
[2024-10-02T18:59:17.183+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-15T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T18:59:17.272+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='carregar' AIRFLOW_CTX_EXECUTION_DATE='2024-01-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-15T00:00:00+00:00'
[2024-10-02T18:59:17.275+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T18:59:17.337+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T18:59:17.337+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T18:59:17.346+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=carregar, run_id=scheduled__2024-01-15T00:00:00+00:00, execution_date=20240115T000000, start_date=20241002T185916, end_date=20241002T185917
[2024-10-02T18:59:17.386+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T18:59:17.402+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-02T18:59:17.404+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-10-02T19:25:35.290+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T19:25:35.481+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-15T00:00:00+00:00 [queued]>
[2024-10-02T19:25:35.487+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-15T00:00:00+00:00 [queued]>
[2024-10-02T19:25:35.488+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T19:25:35.498+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): carregar> on 2024-01-15 00:00:00+00:00
[2024-10-02T19:25:35.503+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=9604) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T19:25:35.504+0000] {standard_task_runner.py:63} INFO - Started process 9606 to run task
[2024-10-02T19:25:35.504+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'carregar', 'scheduled__2024-01-15T00:00:00+00:00', '--job-id', '2298', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpy4ku2fsx']
[2024-10-02T19:25:35.506+0000] {standard_task_runner.py:91} INFO - Job 2298: Subtask carregar
[2024-10-02T19:25:35.540+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-15T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T19:25:35.635+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='carregar' AIRFLOW_CTX_EXECUTION_DATE='2024-01-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-15T00:00:00+00:00'
[2024-10-02T19:25:35.637+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T19:25:35.686+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T19:25:35.687+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T19:25:35.694+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=carregar, run_id=scheduled__2024-01-15T00:00:00+00:00, execution_date=20240115T000000, start_date=20241002T192535, end_date=20241002T192535
[2024-10-02T19:25:35.717+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T19:25:35.731+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-02T19:25:35.733+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
